terraform/domain.tf

```diff
@@ -79,4 +79,12 @@ resource "aws_route53_record" "docs" {
   type    = "CNAME"
   ttl     = "300"
   records = ["lincbrain.github.io."]
+}
+
+resource "aws_route53_record" "dashboard" {
+  zone_id = aws_route53_zone.linc-brain-mit.zone_id
+  name    = "dashboard"
+  type    = "CNAME"
+  ttl     = "300"
+  records = ["lincbrain.github.io."]
```

Comment:
I think this is going to conflict with the `records` section in the `resource "aws_route53_record" "docs"` block above